### DISCUSSION TITLE
How to explore the importance of each feature in node classification

### DISCUSSION THREAD

**Discussion by zhenghaohua11** [pyg-team/pytorch_geometric#7134] [Q&A] [answered]
How to explore the importance of each feature in node classification

**Comment by wsad1** [comment, marked as answer]
Check out the [explainability framework in pyg](https://pytorch-geometric.readthedocs.io/en/latest/tutorial/explain.html). Specifically check out this [example](https://pytorch-geometric.readthedocs.io/en/latest/tutorial/explain.html#explaining-node-classification-on-a-homogeneous-graph) in that doc.

**Reply by zhenghaohua11** [reply]
How can I explore the importance of features across different dimensions in the entire model, rather than just the importance of individual nodes?

**Reply by wsad1** [reply]
Check out this [function](https://pytorch-geometric.readthedocs.io/en/latest/modules/explain.html#torch_geometric.explain.Explanation.visualize_feature_importance), to visualize feature importance.

**Reply by zhenghaohua11** [reply]
Thank you very much for your answer
